dist/.magento.env.yaml

```diff
@@ -261,7 +266,12 @@
 #                  host: mq.host                                                                                      #
 #                  port: 1234                                                                                         #
 #######################################################################################################################
-# SEARCH_CONFIGURATION - use this variable to retain customized search service settings between deployments.          #
+# SEARCH_CONFIGURATION - Replace or modify the Magento search configuration generated by ece-tools during the Cloud   #
+#                        deployment process. The generated configuration is based on the available relationships in   #
+#                        the .magento.app.yaml file.                                                                  #
```

Comment:
> The generated configuration is based on the available relationships in the .magento.app.yaml file.  

Is it true for pro environments? We have environments where services are adding by request, so we actually read relationships from the Environment variable and relationships in  .magento.app.yaml can be not valid.

Reply:
Bleh, you're right. I'll drop out the `.magento.app.yaml` portion; just "available relationships"